nasa/openmct
# New time conductor popups should be closed by hitting ESC key

Addressing issue https://github.com/nasa/openmct/issues/3976

### All Submissions:


### Author Checklist

* [ ] Unit tests included and/or updated with changes? N/A

### Testing instructions:
1. At openmct select realtime mode by switching to 'Local Clock' at time conductor.
2. Click on either start time or end time to open the time input popup.
3. Press the ESC key button and observe that the time input popup is closed now.
